features/step_definitions/helper_services.rb

```diff
@@ -500,9 +500,9 @@
 Given /^I have a iSCSI setup in the environment$/ do
   ensure_admin_tagged
 
-  _project = project("iscsi-target", switch: false)
+  _project = project("openshift-iscsi-target", switch: false)
   if !_project.exists?(user:admin, quiet: true)
-    @result = admin.cli_exec(:create_namespace, name: 'iscsi-target')
+    @result = admin.cli_exec(:create_namespace, name: 'openshift-iscsi-target')
     raise 'failed to "iscsi-target" project' unless @result[:success]
   end
```

Comment:
The cases will still fail once https://github.com/openshift/cluster-kube-apiserver-operator/pull/1369 is merged. That will make "enforce" default to "restricted". Currently your fix can pass because currently the default is still "privileged", which can be checked via `jq "" $(oc extract cm/config -n openshift-kube-apiserver --confirm) | jq '.admission.pluginConfig.PodSecurity'` if you like to check. The thorough fix is still to label your namespace explicitly.

Reply:
Fixed.